src/Inviqa/Worldpay/Api/XmlNodeConverter.php

```diff
@@ -34,7 +34,9 @@ private function convertNode(XmlConvertibleNode $node)
 
             if (!empty($children)) {
                 foreach ($children as $child) {
-                    $this->convertNode($child);
+                    if(null !== $child){
```

Comment:
What happens without the null comparison?

Reply:
Argument passed to converNode has to be XmlConvertibleNode